sockeye/training.py

```diff
@@ -814,8 +815,10 @@ def __call__(self, epoch: int, updates: int, samples: int, tokens: int, metric:
 
         if self.init:
             if count % self.frequency == 0:
-                samples_per_sec = self.samples / (time.time() - self.tic)
-                tokens_per_sec = self.tokens / (time.time() - self.tic)
+                toc = (time.time() - self.tic)
+                updates_per_sec = self.frequency / toc
```

Comment:
do you mean count? frequency stays 50,no?

Reply:
I mean frequency. toc is the time taken for going out of this block and coming back in. Hence frequency/toc is the number of updates/sec